src/components/Button/index.js

```diff
@@ -231,9 +263,52 @@ const StyledButton = styled.button`
     ${props =>
       props.disabled &&
       css`
-        color: ${color.spaceLighter};
-        background-color: ${color.spaceLightest};
-        text-shadow: none;
+        color: rgba(255, 255, 255, 0.6);
+        background-color: ${color.earth};
+      `};
+
+    ${props =>
+      props.variant === 'success' &&
+      props.disabled &&
+      css`
+        background-color: ${color.titan};
+      `};
+
+    ${props =>
+      props.variant === 'warning' &&
+      props.disabled &&
+      css`
+        background-color: ${color.sun};
+      `};
+
+    ${props =>
+      props.variant === 'danger' &&
+      props.disabled &&
+      css`
+        background-color: ${color.mars};
+      `};
+
+    ${props =>
+      props.variant === 'secondary' &&
+      props.disabled &&
+      css`
+        color: ${color.earthLighterAlpha};
+        background-color: #f0fbfe;
```

Comment:
Maybe add a `cursor: no-drop` for the disabled buttons?

Reply:
Removed that in this PR because the cursor feels huge in Chrome. Buttons by default have the `pointer`-cursor, and disabled buttons just won’t change the cursor on hover. It’s more subtle like that.